flang/lib/Lower/OpenMP.cpp

```diff
@@ -110,9 +110,11 @@ genOMP(Fortran::lower::AbstractConverter &converter,
       std::get<Fortran::parser::OmpBeginBlockDirective>(blockConstruct.t);
   const auto &parallelDirective =
       std::get<Fortran::parser::OmpBlockDirective>(blockDirective.t);
-  if (parallelDirective.v == llvm::omp::OMPD_parallel) {
+
     auto &firOpBuilder = converter.getFirOpBuilder();
     auto currentLocation = converter.getCurrentLocation();
+    llvm::ArrayRef<mlir::Type> argTy;
+  if (parallelDirective.v == llvm::omp::OMPD_parallel) {
```

Comment:
Rename with prior suggestion. Otherwise the comparaison looks odd.
```suggestion
  if (blockDirective.v == llvm::omp::OMPD_parallel) {
```